docs/release-notes/release-notes-0.19.0.md

```diff
@@ -136,6 +136,11 @@
   the misnomer of `chan_id` which was describing the short channel
   id to `scid` to represent what it really is.
 
+* [Trigger](https://github.com/lightningnetwork/lnd/pull/9491) the cooperative
+  channel closure even if there are HTLCs active on the channel. LND will block
+  any new HTLCs on the channel and will wait until all HTLCs are resolved to
+  start the cooperative close negotiation.
```

Comment:
It is still not clear to me from the comment near `NoWait: true` if it will wait for HTLCs to be resolved (and removed) and then coop close or initiate a coop close tx with HTLCs as outputs of coop close tx.

The release notes tell that "LND will [...] wait until all HTLCs are resolved to start the cooperative close negotiation", implying the former, while the comment "This makes sure that a coop close will also be executed if active HTLCs are present on the channel." implies the later (or maybe I misunderstand).

Can you elaborate on the change, please? I also propose to update the comment, the commit message and/or release notes to make it clear and consistent.

Reply:
ok refined the release notes.